02-PythonIntro/00. How to run Python code.ipynb

```diff
@@ -21,7 +21,7 @@
     "To run Python code interactively, one can use the standard Python prompt, which can be launched by typing ``python`` in your standard shell:\n",
     "\n",
     "    $ python\n",
-    "    Python 3.5.2 (default, Jul 5 2016, 11:41:13) \n",
+    "    Python 3.11.9 | packaged by conda-forge | (main, Apr 19 2024, 18:34:54) [Clang 16.0.6 ] on darwin\n",
```

Comment:
Thought we want miniforge,  but I see conda-forge here. Are they the same?

Reply:
This will still show up as conda-forge as the package source